pkg/sourcereader/embedded_test.go

```diff
@@ -119,19 +120,20 @@ func (s *MySuite) TestCopyFSToTempDir(c *C) {
 }
 
 func (s *MySuite) TestGetModuleInfo_Embedded(c *C) {
-	ModuleFS = getTestFS()
+	ModuleFS = moduleFS
+	modulereader.ModuleFS = moduleFS
 	reader := EmbeddedSourceReader{}
 
 	// Success
-	moduleInfo, err := reader.GetModuleInfo("modules/network/vpc", tfKindString)
+	moduleInfo, err := reader.GetModuleInfo("modules/test_data/network/vpc", tfKindString)
 	c.Assert(err, IsNil)
 	c.Assert(moduleInfo.Inputs[0].Name, Equals, "test_variable")
 	c.Assert(moduleInfo.Outputs[0].Name, Equals, "test_output")
 
 	// Invalid: No embedded modules
 	badEmbeddedMod := "modules/does/not/exist"
 	moduleInfo, err = reader.GetModuleInfo(badEmbeddedMod, tfKindString)
-	expectedErr := "failed to copy embedded module at .*"
+	expectedErr := "failed to get info using tfconfig for terraform module at modules/does/not/exist: Source to module does not exist: modules/does/not/exist"
```

Comment:
To avoid hard coding the fake path, try using .* instead. The following should be sufficient:
`expectedErr := "failed to get info using tfconfig for terraform module at .*"`

It's enough to ensure the error string is the same, but decreases maintenance since we don't have to update the expected string whenever the test path changes.